### ISSUE TITLE
Fine-tune students with 8-bit

### ISSUE THREAD

**Issue by ZJaume** [mozilla/firefox-translations-training#890]
Fine-tuning the student emulating 8-bits really helps a lot shortening the gap between quantized and non quantized model. The original students pipeline had an example
but it is really just doing an additional step with the same training parameters as a normal training but with a little lower learning rate and `--quantize-bits 8` option. There's also important during this step to use chrf/bleu as main optimization metric or `early-stopping-on: all` option. As it is quite probable CE is going to deteriorate but chrF/BLEU keep improving.

**Comment by eu9ene** [comment]
We already do that https://github.com/mozilla/firefox-translations-training/blob/main/pipeline/train/configs/training/student.finetune.yml. We always use chrF as a main validation metric and ce-mean-words as a cost type. Is that what you mean?

**Comment by ZJaume** [comment]
But is that config being actually used at some point? I could only find it being used here https://github.com/mozilla/firefox-translations-training/blob/b0b5f25d0289a90619a12e645683cfd671332a85/tests/data/experiments_1_12/models/fi-en/opusprod/student-finetuned/train.log#L3

**Comment by eu9ene** [comment]
Yes, it is being used by this step: 

https://github.com/mozilla/firefox-translations-training/blob/b0b5f25d0289a90619a12e645683cfd671332a85/taskcluster/kinds/finetune-student/kind.yml#L104

This training log from tests is likely for testing an offline W&B uploader. Here's a real finetune-student training log: https://firefox-ci-tc.services.mozilla.com/tasks/N8X0cVnzSJufACHEBtK6Zw/runs/0/logs/public/logs/live.log
